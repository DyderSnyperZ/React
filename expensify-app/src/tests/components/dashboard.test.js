import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/dashboard';

test('should be dashboard page', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
