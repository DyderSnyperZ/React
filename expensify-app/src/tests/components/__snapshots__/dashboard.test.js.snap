// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseListe) />
</div>
`;
